src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/SyncCloudContent.targets

```diff
@@ -0,0 +1,31 @@
+<Project ToolsVersion="12.0" DefaultTargets="DownloadBlobsFromAzureTargets" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask TaskName="DownloadFromAzure" AssemblyFile="$(ToolsDir)net45/Microsoft.DotNet.Build.CloudTestTasks.dll"/>
```

Comment:
Ok, that's fine.  I selfishly wanted something I place because I am currently hacking a project with pieces of @maririos change, and I am directly calling this targets file rather than utilizing a wrapping script. Once this is all part of the clean / sync / build workflow, my hack is unnecessary.  I will continue to hack around this in the meantime. :)